packages/cli/src/UserManagement/decorators.ts

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable import/no-cycle */
+import { Application, Request, RequestHandler as Middleware, Response, Router } from 'express';
+import { ResponseHelper } from '..';
+import config from '../../config';
+
+export interface RouteMetadata {
+	method: Method;
+	path: string;
+	handlerName: string | symbol;
+}
+
+const Keys = {
+	ROUTES: Symbol('routes'),
+	BASE_PATH: Symbol('base_path'),
+} as const;
+
+type Method = 'get' | 'post' | 'patch' | 'delete';
```

Comment:
I know we don't need it atm but would also add `put`
```suggestion
type Method = 'get' | 'post' | 'patch' | 'put' | 'delete';
```